transactionaldriver-and-tomcat/pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.narayana</groupId>
+    <artifactId>transactionaldriver-and-tomcat</artifactId>
+    <version>1.0.0</version>
```

Comment:
All the other quickstarts version track narayana master so is 1.0.0 intentional?